library/pk_wrap.c

```diff
@@ -203,7 +203,8 @@ static int rsa_verify_wrap(void *ctx, mbedtls_md_type_t md_alg,
                            const unsigned char *hash, size_t hash_len,
                            const unsigned char *sig, size_t sig_len)
 {
-    mbedtls_rsa_context *rsa = (mbedtls_rsa_context *) ctx;
+    mbedtls_pk_context *pk = (mbedtls_pk_context *) ctx;
```

Comment:
Note: actually I think we could now make the first argument a proper `mbedtls_pk_context *` rather than a `void *` that we then have to cast first thing in every function.

Question: actually could we even make it a `const mbedtls_pk_context *`?